backend/src/routes/chat.js

```diff
@@ -0,0 +1,41 @@
+const express = require('express');
+const router = express.Router();
+
+const messageStore = require('../chat/messageStore');
+
+router.get('/conversations/:conversationId', async (req, res, next) => {
+    try {
+        const messages = await messageStore.getConversation(req.params.conversationId);
+        res.json(messages);
+        next();
+    } catch (error) {
+        res.status(404).json({ error: error });
+        next(error);
+    }
+});
+
+router.get('/conversations/user/:userId', async (req, res, next) => {
+    try {
+        const conversations = await messageStore.getConversationsByUser(req.params.userId);
```

Comment:
Hm @FarihaIS was talking about how she needs the name of the user for each conversation so it can be displayed. So userId is the id of the user whose chatbox we are trying to fetch and each conversation there is with another user who has their own id. Do you think we should package the user first and last name here before we send the response. Or shall we let @FarihaIS make a GET for the user profile based on each conversation?

Reply:
Not sure, we will eventually need the profile picture too right? Then we may as well have them as separate calls because I'm not comfortable putting that info in the response too.

Reply:
Good call. Then @FarihaIS will need to make a separate GET for the user profile.